script.js

```diff
@@ -1 +1,22 @@
-// your code
+// function renderdata
+function renderData(arr){
+    fetch('https://gist.githubusercontent.com/maratgaip/44060c688fcf5f2b7b3985a6d15fdb1d/raw/e93c3dce0826d08c8c6e779cb5e6d9512c8fdced/restaurant-menu.json')
+    .then(res => res.json())
+    .then(data => {
```

Comment:
you supposed to divide this logic from render data.
1.create global object data, so we can reuse this data when you fetch it.
2. create function, which will be fetching data from the server and store this data in global object data.
3. change your renderData function, so we can reuse it